lms/djangoapps/courseware/plugins.py

```diff
@@ -267,3 +267,47 @@ def get_allowed_operations(cls, course_key: CourseKey, user: Optional[User] = No
     @staticmethod
     def legacy_link(course_key: CourseKey):
         return urls.reverse('tabs_handler', kwargs={'course_key_string': course_key})
+
+
+class OpenResponsesApp(CourseApp):
```

Comment:
We might want to change this to OpenResponseAsessmentsApp

Reply:
I didn't want this to confusing with `ora` because it feels too synonymous.